### ISSUE TITLE
Add a index page with links to other pages

### ISSUE THREAD

**Issue by shivanka-thunderbolt** [Nodlik/react-pageflip#3]
is it possible to add a index page at the first page and make those indexed items clickable to navigate to pages ?

**Comment by Nodlik** [comment]
Yes it is possible. But unfortunately it is not possible to use React events. 
One option (not very good) is to use document.addEventListener. For example:
https://gist.github.com/Nodlik/2a559b73e55af74cbb522c35cdc2e761
Supported only "button" and "a" tags

**Comment by shivanka-thunderbolt** [comment]
> Yes it is possible. But unfortunately it is not possible to use React events.
> One option (not very good) is to use document.addEventListener. For example:
> https://gist.github.com/Nodlik/2a559b73e55af74cbb522c35cdc2e761
> Supported only "button" and "a" tags
> 
> Please also update the StPageFlip (https://www.npmjs.com/package/page-flip) library to version 1.1.0

Thank you, it works :)

**Comment by taqi457** [comment]
Hey @Nodlik,
Great work and thanks. Any specific reason that normal React Events can not be used here? 
Thanks Again.
